experiment/selinux/Vagrantfile

```diff
@@ -0,0 +1,20 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+    config.vm.box = "centos/8"
+    config.vm.synced_folder '.', '/vagrant/youki', disabled: false
+
+    config.vm.provider "virtualbox" do |v|
```

Comment:
I couldn't use VirtualBox on my PC, so I was using libvirt instead.
I haven't tried this Vagrantfile (After checking this vagrantfile works for libvirt, updated for virtualbox)

Reply:
Hey, can you try the image `generic/centos8` instead? For some reason the `centos/8` box has issues mounting the youki folder on my machine, while the generic one works.

Reply:
Hi @YJDoc2 
`generic/centos8` works on my env (libvirt) as well.
Thanks for your investigation.